### PR TITLE
Airdrop cli

### PR DESCRIPTION
**Pull request by 999max**

### CODE REVIEW

File: airdrop.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    print(f"Token balance:  {config.token_balance} Wei")
+    print(f"Sender nonce: {config.current_nonce}")
+    print(f"Web3 endpoint: {config.web3_node}")
+    print(f"Gas Price: {config.gas_price}\n")
+    
+    print(f"Recipients:")
+    token_sum = Recipient.select(fn.SUM(Recipient.amount))[0].amount
+    recipients = Recipient.select()
+    
+    header = [['Address', 'Tokens', 'Nonce', 'Status']]
```
Comment (ongrid) [inline airdrop.py]: add Txn

File: airdrop.py [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+
+def update_data():
+    config = Config.get(1)
+    # balances
+    w3 = Web3(Web3.HTTPProvider(config.web3_node, request_kwargs={"timeout": 20}))
+    config.eth_balance = w3.eth.get_balance(config.address)
+    with open("tokens_abi/ERC20.abi", "r") as file:
+        erc20_abi = json.load(file)
+    token_contract = w3.eth.contract(address=Web3.toChecksumAddress("0x688ce8a97d5f1193261DB2271f542193D1dFd866"), abi=erc20_abi)
```
Comment (ongrid) [inline airdrop.py]: use cli command to set token address

**Review by ongrid — CHANGES_REQUESTED** [review]
README.md should explain available commands

File: test_airdrop.py [hunk 3]
```diff
@@ -0,0 +1,365 @@
+import peewee as pw
+import airdrop as ad
+from models import BaseModel, Config, Recipient, Tx
+
+from ast import literal_eval
```
Comment (ongrid) [inline test_airdrop.py]: use json 
https://docs.python.org/3/library/json.html

File: test_airdrop.py [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+
+
+# Note.
+# All of addresses are valid the for binance testnet (97). 
+# But no one transaction will happend due to using pytest monkeypatch in 
+# the methods send (tranfer) and get receipts.
+
+
+DBFILE = "db_test.sqlite"
```
Comment (ongrid) [inline test_airdrop.py:13]: :memory:

https://docs.peewee-orm.com/en/latest/peewee/database.html#using-sqlite

**Review by ongrid — CHANGES_REQUESTED** [review]

File: test_airdrop.py (hunk 3 above)
Comment (999max) [reply test_airdrop.py]: fixed

**Review by ongrid — APPROVED** [review]

**Pull request merged**